packages/destination-actions/src/destinations/close/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import createUpdateContactAndLead from './createUpdateContactAndLead'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Close',
+  slug: 'close',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'basic',
+    fields: {
+      api_key: {
+        label: 'Api key',
+        description: 'Your Close api key.',
+        type: 'string',
+        required: true
+      },
+      contact_custom_field_id_for_user_id: {
+        label: 'Contact Custom Field ID for User ID',
```

Comment:
this goes along with my questions about the meaning of "Contact External ID" but is "User ID" the only thing ever stored in this field?

Reply:
Yes